integration-tests/api/__tests__/admin/product.js

```diff
@@ -1410,7 +1424,7 @@ describe("/admin/products", () => {
       expect(response.data.product).toEqual(
         expect.objectContaining({
           title: "Test product",
-          variants: [
+          variants: expect.arrayContaining([
```

Comment:
todo: rm, this is intentionally not `expect.arrayContaining` due to variant ordering

Reply:
+1

Reply:
Ah, fair point. This one was randomly flaking on me.

Reply:
We might need to investigate this test. I've experienced the flakiness as well

Reply:
It is not flacky anymore in the typeorm or 💪🥰